entrypoint.sh

```diff
@@ -1,19 +1,30 @@
-#!/bin/sh
-set -e # Exit immediately if a command exits with a non-zero status
+#!/bin/bash
+set -eu # Increase bash error strictness
 
-if [ -n "${GITHUB_WORKSPACE}" ] ; then
+if [[ -n "${GITHUB_WORKSPACE}" ]]; then
   cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit
 fi
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-remark --frail --quiet --use=remark-preset-lint-recommended . ${INPUT_REMARK_FLAGS} 2>&1 |
+# NOTE: Used for debugging
+if [[ "$#" -eq 0 && "${INPUT_REMARK_FLAGS}" != "" ]]; then
+  remark_args=(${INPUT_REMARK_FLAGS})
+elif [[ "$#" -ne 0 && "${INPUT_REMARK_FLAGS}" != "" ]]; then
+  remark_args=($* ${INPUT_REMARK_FLAGS})
+elif [[ "$#" -ne 0 && "${INPUT_REMARK_FLAGS}" == "" ]]; then
+  remark_args=($*)
+fi
+
+# NOTE: ${VAR,,} Is bash 4.0 syntax to make strings lowercase.
+echo "[action-remark-lint] Checking markdown code with the remark-lint linter and reviewdog..."
+remark --use=remark-preset-lint-recommended . ${remark_args[@]} 2>&1 |
   sed 's/\x1b\[[0-9;]*m//g' | # Removes ansi codes see https://github.com/reviewdog/errorformat/issues/51
   reviewdog -f=remark-lint \
-  -name="${INPUT_TOOL_NAME}" \
-  -reporter="${INPUT_REPORTER:-github-pr-check}"\
-  -filter-mode="${INPUT_FILTER_MODE}" \
-  -fail-on-error="${INPUT_FAIL_ON_ERROR}" \
-  -level="${INPUT_LEVEL}" \
-  -tee \
-  ${INPUT_REVIEWDOG_FLAGS}
\ No newline at end of file
+    -name="${INPUT_TOOL_NAME}" \
+    -reporter="${INPUT_REPORTER,,}" \
```

Comment:
Ditto for other flags.

Reply:
Good point. I removed the lowercase syntax all together as even with fail-on-error this might not work if we change the fail-on-error behaviour in the future.